.github/release-drafter.yml

```diff
@@ -24,6 +23,9 @@ categories:
   - title: '🚩 Requires settings change'
     labels:
       - 'settings_changes'
+  - title: '🚩 Security'
```

Comment:
Please make sure to rebase and not pick any commit not related to your PR. Cheers!